### ISSUE TITLE
Enhancement: Make it easier to pause active scan, skip test categories

### ISSUE THREAD

**Issue by davewichers** [zaproxy/zaproxy#3454]
When you are on the Scan Progress dialog of an active scan, it lists a row for each scan type to run (or that has run), but only has a Skip button for the current scan type being run. I'd like to see 2 simple enhancements:

1) There be a pause button in addition to a skip button for the current test category.
2) There be a skip button for every test category that has not run yet so you can skip individual test categories that have not run yet. (In addition to still being able to skip the currently running test category).

**Comment by thc202** [comment]
Split the skip functionality into other issue #3739.

Could you provide more details on the pause functionality? Do you want the same behaviour as the main pause button (but being in the Scan Progress dialogue)?

**Comment by davewichers** [comment]
Yes. I simply want the pause button to also be displayed on the current row
where the scan is active, to make it easier to find/use. But it should
behave exactly the same, i.e., pause the current scan. And then be a play
button to continue the scan if its paused.

**Comment by thc202** [comment]
Thanks for the explanation!
